index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Baraka Danny || Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <nav class="nav">
+            <a href="logo-link">
+                <h1 class="logo-text">danny</h1>
+            </a>
+            <div class="nav-toggle">
+                <img src="img/toggle.png" alt="">
+            </div>
+            <div class="nav-menu">
+                <ul>
+                    <li class="nav-item">Portfolio</li>
+                    <li class="nav-item">About</li>
+                    <li class="nav-item">Contact</li>
+                </ul>
+            </div>
+        </nav>
+    </header>
+
+    <main class="main">
+        <section class="hero" id="hero">
+            <!-- background graphics -->
+            <img class="graph graph-1" src="img/graph_1.png" alt="graph-1">
+            <img class="graph graph-2" src="img/graph_2.png" alt="graph-2">
+            <img class="graph graph-3" src="img/graph_3.png" alt="graph-3">
+            <img class="graph graph-4" src="img/graph_4.png" alt="graph-4">
+            <img class="graph graph-5" src="img/graph_5.png" alt="graph-5">
```

Comment:
Great Job so far @barakadanny.🤝. 

- [ ] Try to follow the Figma design provided in the project requirements
- Figma Design
![Figma design](https://user-images.githubusercontent.com/63917725/176554032-7e9fa466-b597-4ed4-9b10-16b99af369dc.PNG)
- Your design
![Your design](https://user-images.githubusercontent.com/63917725/176554122-96a46978-09a2-4847-8e6a-f25ead56e8bc.PNG)